lib/models/project.js

```diff
@@ -323,7 +329,7 @@ class Project {
    */
   resolveSync(file) {
     return resolve.sync(file, {
-      basedir: this.root,
+      basedir: process.env.EMBER_NODE_PATH || this.root,
```

Comment:
I don't grok what this environment variable is about, can you explain?

Reply:
This is only used in one addon (per https://emberobserver.com/code-search?codeQuery=EMBER_NODE_PATH) and that looks like a copy-paste of ember-cli's own tests?

Reply:
I would be totally happy to eliminate this feature, I just can't tell who's depending on it.  I think it's more of an app feature than an addon feature.

It was added here: https://github.com/ember-cli/ember-cli/pull/3771

I think it was a roundabout way of doing what this PR is also doing: allow people to put their node_modules in the other places that you would expect to work based on node behavior.

Reply:
Yeah, agreed.  Would you mind triggering a deprecation if we detect that `process.env.EMBER_NODE_PATH` is set?

Reply:
Lets take this as a separate issue (and not block this PR on it)...